pkg/kubefedctl/enable/validation.go

```diff
@@ -137,6 +138,11 @@ func federatedTypeValidationSchema(templateSchema map[string]v1beta1.JSONSchemaP
 												},
 											},
 										},
+										// TODO(marun) Enable defaulting for federated types
```

Comment:
I don't think enabling defaulting for federated types should constitute making `op` required, as there will be CRD client-side validation that would still fail. However, we can still make it required in validation while still leaving the field optional.

Reply:
Fair point, I'll remove the comment. I find this behavior very counter-intuitive, and question whether client-side validation is more a blessing or a curse.